package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "github-canned-responses",
+  "version": "1.0.5",
+  "description": "Choose from a set of canned responses when commenting on your GitHub PRs or issues",
+  "main": "background.js",
+  "dependencies": {
+    "gulp-json-editor": "^2.2.1",
```

Comment:
Suggestion: Add `gulp` as a dependency, and add an npm "run" script that executes `gulp`. It will remove the requirement for users to have the Gulp CLI installed globally